tests/functional/conftest.py

```diff
@@ -28,3 +31,75 @@ def statsd_exporter_image(metadata):
 def influx_model_name():
     """Get influx's model name for testing."""
     return "influxdbmodel"
+
+
+@pytest_asyncio.fixture(scope="module")
+async def app(
+    ops_test: OpsTest,
+    gunicorn_image: str,
+    statsd_exporter_image: str,
+    influx_model_name: str,
+):
+    """
+    arrange: given a development version of the gunicorn charm
```

Comment:
This is somewhat picky, since this is now a fixture (thanks for converting it), the docstring should just be a description of the fixture rather than the test arrange, act and assert since a fixture should only do asserts related to ensuring that the state is correctly configured rather than any that check for behaviour that would be checked for in a test